geo/tsconfig.json

```diff
@@ -10,14 +10,20 @@
     "sourceMap": true,
     "outDir": "./dist",
     "baseUrl": "./",
+    "lib": ["es2021", "dom"],
     "incremental": true,
     "skipLibCheck": true,
     "strict": true,
     "strictNullChecks": false,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

Comment:
We should keep this option being true. I know ported code is working with different rule, but using this option help us in the future.